manifests/a/AGFEO/AGFEODashboard/2.2.0/AGFEO.AGFEODashboard.locale.de-DE.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.3.2 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: AGFEO.AGFEODashboard
+PackageVersion: 2.2.0
+PackageLocale: de-DE
+Publisher: AGFEO
+PublisherUrl: https://agfeo.de
+# PublisherSupportUrl:
+PrivacyUrl: https://agfeo.de/datenschutz/
+Author: AGFEO GmbH & Co. KG
+PackageName: AGFEO Dashboard
+# PackageUrl:
+License: Proprietary
+# LicenseUrl:
+Copyright: 2024, AGFEO GmbH & Co. KG
+# CopyrightUrl:
+ShortDescription: Dashboard for use with AGFEO Phone Systems
```

Comment:
This should ideally be translated in German as it's a `de-DE` locale file

Reply:
I changed the language of the `ShortDescription` to German